src/main/scala/strawman/collection/mutable/Seq.scala

```diff
@@ -0,0 +1,76 @@
+package strawman
+package collection
+package mutable
+
+trait Seq[A] extends strawman.collection.Seq[A] with Growable[A] {
```

Comment:
This is what's wrong:

    scala> collection.mutable.Seq(1, 2, 3)
    res1: scala.collection.mutable.Seq[Int] = ArrayBuffer(1, 2, 3)

So far, even the default `Seq` naturally supports `+=`. I don't think we can take that away. So the less drastic change would probably be to have a `FixedLengthSeq` superclass of `Seq` and have `WrappedArray` inherit from that instead of `Seq`.